### DISCUSSION TITLE
Add command line tool to get events count

### DISCUSSION THREAD

**Discussion by caffeine-mgn** [d99kris/nchat#183] [Ideas]
Hi. It will be great if `nchat` got a tool for getting events count.
For example, I can use this cmd for i3/awesome/sway for drawing events counter on top panel.
Something like this: `nchat --events-counter [profile]` and then `nchat` returns `5` and `nchat` closing after this response.

**Comment by d99kris** [comment]
Hi @caffeine-mgn - thanks for reporting a feature request. This requires similar backend fixes as #154. Probably can revisit this request once that feature request has been addressed.
